resources/css/theme.css

```diff
@@ -199,7 +199,8 @@ html[dir="rtl"] .hamburger-menu:before {
     }
 
     form .w-1\/2 > div.flex {
-        display: block;
+       display: flex;
+       flex-wrap: nowrap;
     }
```

Comment:
would be better to remove this code instead of creating another useless override